### PR TITLE
Issue 39 Service/subservice instead of tenant/service

### LINKED ISSUE #39: Unify usage of "service" and "subservice" in all the code
(Raised in https://github.com/telefonicaid/perseo-fe/pull/37)

We should rationalize the usage of the service and subservice concetps in the code. Currently, there some problems:

* The usage of old term "tenant" for service
* The usage of "service" in places where actually it means "subservice"

So, a overall review of the code is needed to fix all this things and make code more maintenable.

**Effort: 2 man day**

### PR DESCRIPTION
**Pull request by crbrox**
Fix #39 

@fgalan 
@dmoranj 
@jcalderin

### CODE REVIEW

**Comment by fgalan** [comment]
LGTM (passing the ball to @dmoranj for final LGTM + merge + delete branch)

File: test/utils/clients.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ function commonRequest(method, url, object, callback) {
     var options = {};
     options.headers = {};
     options.headers[constants.TENANT_HEADER] = config.DEFAULT_TENANT;
```
Comment (jcalderin) [inline test/utils/clients.js]: Is this constants still available? (TENANT Reference one)

Comment (crbrox) [inline test/utils/clients.js]: No
fixed in 6dc5c9a

**Comment by dmoranj** [comment]
LGTM

**Comment by jcalderin** [comment]
LGTM

**Pull request merged**